front/app/containers/Admin/messaging/AutomatedEmails/index.tsx

```diff
@@ -76,7 +76,7 @@ class AutomatedCampaigns extends React.PureComponent<
 
 const AutomatedCampaignsWithIntl = injectIntl<Props>(AutomatedCampaigns);
 
-export default (inputProps: Props) => (
```

Comment:
Why would you remove the typing?

Reply:
In pairing with @IvaKop we figured that these InputProps aren't actually used. The component itself is still fully typed (`campaign` is the only prop and it's correct) but the input props concept I believe is copied/pasted from other components and not actually used here, in that we don't actually pass any props in to the surrounding data component. The only prop here is `campaign` and that comes from the HOC, not from being passed in as React props at any point. 

We did the same in some of the other files, but I removed the typing here because it was complaining. I guess the actual solution would be that `inputProps` can be removed entirely.

@IvaKop correct me if I'm getting something wrong here 😅

Reply:
Yeah sure, no need to dive in @IvaKop. We'll figure it out. :-)

Reply:
Yeah, before we were using `loadable` so I think Typescript didn't check to make sure the props were there, but now with React.lazy and calling the component directly like `<AutomatedEmailsIndex />` it expects those props to be passed in as actual props if they're typed in that place. If the typing on the HOC is removed then you can invoke the component as a component normally and it will remain typed internally, but doesn't need those props passed in as such